app/rdf/query-cube-metadata.ts

```diff
@@ -42,11 +49,41 @@ export const createThemeLoader =
     }
 
     const parsed = results.map(parseSparqlTheme);
+    return parsed;
+  };
+
+export const createOrganizationLoader =
+  ({ locale }: { locale: string }) =>
+  async (
+    filterIris?: readonly string[]
+  ): Promise<(DataCubeOrganization | null)[]> => {
+    const query = SELECT.ALL.WHERE`
+    graph   <https://lindas.admin.ch/sfa/opendataswiss> {
+      ?theme a ${schema.Organization} ;
+        ${schema.name} ?name ;
+      filter (langmatches(lang(?name), "${locale}"))
```

Comment:
Maybe not relevant to the prototype but have a look at the `loadResourceLabels` function from app/rdf/query-labels.ts here. What that does is it falls back to other languages if the main one is not available.

Reply:
Ah yes, I had seen that and moved it to the end of the prototype, I will take a look thanks.

Reply:
Done in https://github.com/visualize-admin/visualization-tool/pull/199/commits/633f2b1c772eafee69dbdbf412cebcb08a97e095